app/assets/javascripts/discourse/app/lib/plugin-api.gjs

```diff
@@ -327,6 +331,29 @@ class PluginApi {
     return klass;
   }
 
+  /**
+   * Add a new valid transformer name.
+   *
+   * Use this API to add a new transformer name that can be used in the `registerTransformer` API.
+   *
+   * @param name the name of the new transformer
+   */
+  addTransformerName(name) {
+    addTransformerName(name);
+  }
+
+  /**
+   * Register a transformer to override values defined in Discourse.
+   *
+   * @param {string} transformerName the name of the transformer
+   * @param {*} valueOrCallback new value or callback to be used to transform the value, if a callback is provided it
+   * will be called with ({value, context}), where value is the current value and context is the context in which the
+   * value is being used.
```

Comment:
Not sure about offering two different systems here. It'll be much easier to document/explain if there is only one choice.

If people only want a simple value, they can do this with only 4 more characters:

```
registerTransformer("blah", () => "simple value")
```

Reply:
Done.